### PR TITLE
Add support for Heroku Redis

### PR DESCRIPTION
**Pull request by mattstibbs**
Added support for Heroku Redis which sets an environment variable of REDIS_URL.

Not sure if I've missed some implication of simply adding like this but worth a pull request!

### CODE REVIEW

**Comment by singingwolfboy** [comment]
This looks good to me! Can you actually put the Heroku Redis section before Redis To Go? I think it makes more sense that way.

**Comment by mattstibbs** [comment]
@singingwolfboy Thanks - have swapped those sections around now. 

Is it ok that for someone using both Heroku Redis and Redis To Go, Redis To Go would take precedence? Not that I can think of a different way of handling someone using both add-ons at the same time :)

**Comment by singingwolfboy** [comment]
Actually, because you're using `.setdefault()`, the *first* section will take preference, not the last. That's one of the reasons I suggested you switch them. :smile:

File: flask_heroku.py [hunk 1]
```diff
@@ -66,7 +66,15 @@ def init_app(self, app):
...
-
+            
+        # Heroku Redis
+        redis_url = environ.get('REDIS_URL')
+        if redis_url:
+            url = urlparse(redis_url)
+            app.config.setdefault('REDIS_HOST', url.hostname)
+            app.config.setdefault('REDIS_PORT', url.port)
+            app.config.setdefault('REDIS_PASSWORD', url.password)
+
```
Comment (singingwolfboy) [inline flask_heroku.py]: Please remove trailing whitespace from this line.

File: flask_heroku.py [hunk 2]
```diff
@@ -66,7 +66,15 @@ def init_app(self, app):
             app.config.setdefault('MAIL_USERNAME', environ.get('POSTMARK_API_KEY'))
             app.config.setdefault('MAIL_PASSWORD', environ.get('POSTMARK_API_KEY'))
             app.config.setdefault('MAIL_USE_TLS', True)
-
+
```
Comment (singingwolfboy) [inline flask_heroku.py]: Please remove trailing whitespace from this line.

**Comment by mattstibbs** [comment]
@singingwolfboy Thanks - whitespace removed (and I'll watch out for that in future)

**Comment by singingwolfboy** [comment]
@mattstibbs You missed a line :)

**Comment by mattstibbs** [comment]
@singingwolfboy Ah damn. Think I've got it now? (thanks for being patient!)

**Comment by singingwolfboy** [comment]
Looks great! I'll merge this now. I'll wait a few days to see if @brainix is going to submit another PR (as he said he would in #27). Then, I'll cut a new release and push it to PyPI.

**Comment by brainix** [comment]
I apologize but I didn't have a chance to get to my PR yet.  I'm still going to create a new PR - and I'll do it ASAP, as I know that the release is waiting!  Thanks.

**Comment by singingwolfboy** [comment]
@brainix: no worries, I know how life can get in the way. :smile: There's no hurry. If I don't get a PR from you in a few days, I'll cut a release without it, but we can always make another release after your PR gets in.

**Comment by brainix** [comment]
Thanks for understanding!

**Pull request merged**